src/components/TodoComponents/TodoList.js

```diff
@@ -1,2 +1,45 @@
 // your components will all go in this `component` directory.
 // feel free to change this component.js into TodoList.js
+import React, { Component } from 'react'
+import Todo from './Todo'
+import TodoForm from './TodoForm'
+import './Todo.css'
+
+export default class TodoList extends Component {
+    constructor(props){
+        super(props)
+        this.checkmark=this.checkmark.bind(this)
+        this.add=this.add.bind(this)
+        this.clear=this.clear.bind(this)
+    }
```

Comment:
Same note as before.

I like the separation of concerns though, especially on bigger projects but this is to guide students through prop drilling, and recognizing the value of a centralized state and single source of truth